pipeline/inputs/windows-exporter-metrics.md

```diff
@@ -90,6 +95,87 @@ You can test the expose of the metrics by using _curl:_
 curl http://127.0.0.1:2021/metrics
 ```
 
+### Service where clause
+
+Windows service collector will retrieve entire service information in the local box.
+`we.service.where`, `we.service.include`, and `we.service.exclude` can operate to reduce the volume of the service metrics.
+
+To reduce the amount of this metrics, users have to use where clause.
```

Comment:
I think we should uppercase WHERE, it's not the normal word but a specific syntax?

Reply:
Right. I used upper case for where clause description here.